courses/migrations/0037_update_programrequirements_add_elective_flag.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.18 on 2023-07-10 15:41
+
+from django.db import migrations, models
+
+
+def forwards_func(apps, schema_editor):
+    ProgramRequirement = apps.get_model("courses", "ProgramRequirement")
+    db_alias = schema_editor.connection.alias
+    update_reqs = (
+        ProgramRequirement.objects.using(db_alias)
+        .filter(depth=2, title="Elective Courses")
```

Comment:
Should `elective_flag = True` also be set for nested elective operators?

Reply:
No, it's really only meant to count if `depth=2` (1 being the root node for the tree). It doesn't have any meaning past that.